interface-definitions/protocols-pimv6.xml.in

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0"?>
+<!-- Protocol Independent Multicast for IPv6 (PIMv6) configuration -->
+<interfaceDefinition>
+  <node name="protocols">
+    <children>
+      <node name="pimv6" owner="${vyos_conf_scripts_dir}/protocols_pimv6.py">
```

Comment:
Please use only `pim6` as FRR has `pim` and `pim6` no need for an extra `v`

Reply:
Done